python/grass/gunittest/gutils.py

```diff
@@ -39,9 +39,9 @@ def is_map_in_mapset(name, type, mapset=None):
     # so anything accepted by g.findfile will work but this can change in the
     # future (the documentation is clear about what's legal)
     # supporting both short and full names
-    if type == "rast" or type == "raster":
+    if type in {"rast", "raster"}:
```

Comment:
This whole expression is probably more elegantly solved with a dict:

```
type = {
"rast": "cell",
"raster": "cell",
"rast3d": "grid3",
"raster3d": "grid3",
"vect": "vector"}[type]
```

Reply:
I agree, but since it is in gunittest, I think I'd want to do that change only once I have set up code coverage for the non-pytest parts of the code. It's what is keeping us "safe" during that refactor.

I'm keeping it in mind.

Reply:
Also, some of these patterns would be nice (mostly other where in the code) with some typing annotations. I explored it a bit further with one file, a first PR will be coming at some point. I also saw a couple places where some trivial typings could be added easily.